src/helpers/validate.ts

```diff
@@ -14,10 +14,15 @@ export function validateURL(url: string): boolean {
   return validator.isURL(url, { require_protocol: true })
 }
 
-export function validateFlags(flags: string): boolean {
+export function validateFlags(flag: string): boolean {
   // eslint-disable-next-line no-useless-escape
   const mask = /^[\w\.\-]{1,45}$/
```

Comment:
```suggestion
```

Reply:
@mitchell-codecov Why are you deleting the mask?